### PR TITLE
TFKUBE-335: Make Bitbucket version configurable.

### PR DESCRIPTION
**Pull request by nghazali**

### CODE REVIEW

File: config.tfvars [hunk 1]
```diff
@@ -102,6 +102,10 @@ confluence_db_iops                 = 1000
 # Helm chart version of Bitbucket
 bitbucket_helm_chart_version = "1.2.0"
 
+# By default, Bitbucket will use the version defined in the helm chart. If you wish to override the version, uncomment
```
Comment (yzha645) [inline config.tfvars:105]:
```suggestion
# By default, Bitbucket will use the version defined in the Helm chart. If you wish to override the version, uncomment
```

**Review by yzha645 — APPROVED** [review]

File: config.tfvars [hunk 2]
```diff
@@ -102,6 +102,10 @@ confluence_db_iops                 = 1000
...
+# the following line and set the bitbucket_software_version to any of the versions available on https://hub.docker.com/r/atlassian/bitbucket/tags
```
Comment (yzha645) [inline config.tfvars:106]:
```suggestion
# the following line and set the bitbucket_version_tag to any of the versions available on https://hub.docker.com/r/atlassian/bitbucket/tags
```

**Comment by yzha645** [comment]
The same typo within the comments apply to the other two PRs for Confluence and Bamboo.

**Review by yzha645 — CHANGES_REQUESTED** [review]
The same typo within the comments apply to the other two PRs for Confluence and Bamboo.

**Review by yzha645 — APPROVED** [review]

File: config.tfvars [hunk 3]
```diff
@@ -102,6 +102,10 @@ confluence_db_iops                 = 1000
 # Helm chart version of Bitbucket
 bitbucket_helm_chart_version = "1.2.0"
 
+# By default, Bitbucket will use the version defined in the Helm chart. If you wish to override the version, uncomment
+# the following line and set the bitbucket_version_tag to any of the versions available on https://hub.docker.com/r/atlassian/bitbucket/tags
+#bitbucket_version_tag = "<BITBUCKET_VERSION_TAG>"
```
Comment (errcode1202) [inline config.tfvars:107]:
```suggestion
# By default, Bitbucket will use the version defined in the Bitbucket Helm chart: 
# https://github.com/atlassian/data-center-helm-charts/blob/main/src/main/charts/bitbucket/Chart.yaml 
# If you wish to override the version, uncomment the following line and set the bitbucket_version_tag to any of the # versions published for Bitbucket on Docker Hub: https://hub.docker.com/r/atlassian/bitbucket/tags
#bitbucket_version_tag = "<BITBUCKET_VERSION_TAG>"
```

**Review by errcode1202 — APPROVED** [review]

**Review by nanux — APPROVED** [review]
Pending conflict resolution

**Pull request merged**